src/systems/sensors/Sensors.cc

```diff
@@ -32,19 +32,23 @@
 
 #include <ignition/rendering/Scene.hh>
 #include <ignition/sensors/CameraSensor.hh>
+#include <ignition/sensors/BoundingBoxCameraSensor.hh>
 #include <ignition/sensors/DepthCameraSensor.hh>
 #include <ignition/sensors/GpuLidarSensor.hh>
 #include <ignition/sensors/RenderingSensor.hh>
 #include <ignition/sensors/RgbdCameraSensor.hh>
 #include <ignition/sensors/ThermalCameraSensor.hh>
+// #include <ignition/sensors/SegmentationCameraSensor.hh>
```

Comment:
I know that you commented this out so that you could build this PR with just the Bounding Box PRs in `ign-rendering`/`ign-sensors`, but can you go ahead and uncomment this line? In the PR itself, we should have everything uncommented to avoid accidentally merging code with missing functionality. If you need to comment out certain things for testing a particular sensor locally, that's fine :+1: but, I think we should avoid having these comments in the PR itself